tools/test.py

```diff
@@ -126,6 +126,12 @@
         all_tests = {}
         tests = {}
 
+        # Add the test macro, when compiling all test
+        if options.macros is None:
+           options.macros = (['MBED_TEST'])
```

Comment:
Please remove unneeded parenthesis `(`,`)`

Reply:
Fixed it on dependent PR #7627